src/components/shared/developerCard/index.tsx

```diff
@@ -33,15 +33,17 @@ const DeveloperCardItem = ({
           </Grid>
           <Grid item xs={9}>
             <Typography variant="body1" component="p" noWrap={true}>
-              {oneliner}
+              {oneLiner}
             </Typography>
             <Typography variant="body1" component="p">
               {name}
             </Typography>
             <ChipGroup>
-              {stacks.slice(0, 3).map((stack) => (
-                <ChipStyled key={stack} label={stack} />
-              ))}
+              {techStack &&
+                Array.isArray(techStack) &&
+                techStack.map((stack) => (
+                  <ChipStyled key={stack} label={stack} />
+                ))}
```

Comment:
techStack 은 어차피 배열이니까, 이렇게 줄일 수도 있을 것 같아요!
```suggestion
              {techStack?.map((stack) => (
                  <ChipStyled key={stack} label={stack} />
                )) ?? []}
```